Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    G_addNICs = nx.from_pandas_edgelist(vm_nic_df, "nic", "name")
</pre><p>A couple of thoughts:</p><ol><li>You could get rid of the intermediate G_* variables by just doing the nx.from_pandas_edgelist() inline (e.g. G.add_nodes_from(nx.from_pandas_edgelist(....))</li><li>You could make this even nicer by putting all the variable items in a list of tuples and running the G.add_nodes_from in a for loop.</li><li>Although nx uses uppercase G, etc. Python folks normally keep uppercase only for names of classes or ALL_UPPERCASE for constants</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>done!</p>